spec/src/main/asciidoc/WebSocket.adoc

```diff
@@ -1490,14 +1490,31 @@ either to the *GOLD_MEMBER* or *PLATINUM_MEMBER* roles.
 This appendix lists the changes in the WebSocket specification.
 This appendix is non-normative.
 
+=== Changes Between 2.1 and 2.0
+
+* https://github.com/eclipse-ee4j/websocket-api/issues/190[Issue 190] and
```

Comment:
The biggest change is #211/#375

Reply:
ACK. I was working from some notes I made as I went along. I'll go back and check the commit log and ensure this and any other missing changes are added.